CONTRIBUTING.md

```diff
@@ -789,6 +789,12 @@ setting `USE_PRECOMPILED_HEADERS=1` either on first setup, or in the
 USE_PRECOMPILED_HEADERS=1 python setup.py develop
 ```
 
+If you are using MacOS on Apple M1:
+
+```sh
+USE_PRECOMPILED_HEADERS=1 USE_MPS=1 python setup.py develop
```

Comment:
One does not need to pass `USE_MPS` explicitly, it's enabled by default if OS supports it.
```suggestion
USE_PRECOMPILED_HEADERS=1 python setup.py develop
```